package-lock.json

```diff
@@ -3688,4 +3688,4 @@
       }
     }
   }
-}
+}
```

Comment:
Do you mind undo this change? There should be no diff in `package-lock.json`.

Reply:
Second time... Fixed it now. Sorry!